routes/web.php

```diff
@@ -18,3 +21,12 @@
 Auth::routes();
 
 Route::get('/home', 'HomeController@index')->name('home');
+
+Route::view('add','home');
+Route::post('add',[PostsController::class,'addData']);
+
+Route::get('image-upload', [ ImageUploadController::class, 'imageUpload' ])->name('image.upload');
+Route::post('image-upload', [ ImageUploadController::class, 'imageUploadPost' ])->name('image.upload.post');
+
+Route::get('user/create', [ HomeController::class, 'create' ]);
+Route::post('user/create', [ HomeController::class, 'store' ]);
```

Comment:
See https://laravel.com/docs/master/controllers#resource-controllers for resource route conventions.